Cogs.Tests/CsSchemaTests.cs

```diff
@@ -32,23 +39,34 @@ public void CsForHamburgersTest()
             var publisher = new CsSchemaPublisher();
             publisher.TargetDirectory = outputPath;
             publisher.Publish(cogsModel);
+
+            //set up executable
+            using (Stream resFilestream = Assembly.GetExecutingAssembly().GetManifestResourceStream("Cogs.Tests.dotnet.exe"))
```

Comment:
Just like with dot, dotnet can be run from the environment.
Cogs runs on windows, mac, and linux and dot it will be different per platform.
For the tests, we can assume that it is in the environmental Path on the machine, and add it to our development machines